common/page.go

```diff
@@ -723,6 +743,12 @@ func (p *Page) Reload(opts goja.Value) api.Response {
 		req.responseMu.RUnlock()
 	}
 
+	select {
+	case <-lifecycleEvtCh:
+	case <-timeoutCtx.Done():
+		k6ext.Panic(p.ctx, "%w", wrapTimeoutError(timeoutCtx.Err()))
+	}
+
 	applySlowMo(p.ctx)
```

Comment:
Do we need this slow mo here?

Reply:
Yes, so we can slow down the page reload process.